### PR TITLE
There are only Camel/Kafka sources of interest, and those got relocated

### PR DESCRIPTION
**Pull request by matzew**
changes to reflect the new location of Camel / Kafka,  and generating _separate_ release yaml files

this is required for 0.7.1 ....

### CODE REVIEW

**Comment by matzew** [comment]
@nicolaferraro please check !

File: openshift/release/generate-release.sh [hunk 1]
```diff
@@ -2,25 +2,20 @@
...
+# fi
 
-# Apache Camel-K Source
-resolve_resources contrib/camel/config/ kamel-resolved.yaml $image_prefix $release
-cat kamel-resolved.yaml >> $output_file
+# resolve_resources config/ $output_file $image_prefix $release
 
-# Apache Kafka Source
-resolve_resources contrib/kafka/config/ kafka-resolved.yaml $image_prefix $release
-cat kafka-resolved.yaml >> $output_file
```
[the remark this answers is not in the document]
Comment (matzew) [reply openshift/release/generate-release.sh]: to have it, as reference body for the future

**Comment by nicolaferraro** [comment]
Thanks @matzew !

**Pull request merged**